src/layer.rs

```diff
@@ -140,6 +154,62 @@ impl Layer {
     }
 }
 
+/// The contents of the [`layerinfo.plist`] file.
+///
+/// [`layerinfo.plist`]: https://unifiedfontobject.org/versions/ufo3/glyphs/layerinfo.plist/
+#[derive(Debug, Clone, Default, PartialEq)]
+pub struct LayerInfoData {
```

Comment:
I'd just call this `LayerInfo`.

Reply:
There is a `LayerInfo` elsewhere already 🤔